modules/computation/batch.tf

```diff
@@ -83,5 +83,11 @@ resource "aws_batch_job_queue" "this" {
     aws_batch_compute_environment.this.arn
   ]
 
+  job_state_time_limit_action {
+    action= "CANCEL"
+    max_time_seconds = var.job_state_time_limit_timeout 
+    reason=var.job_state_time_limit_reason
```

Comment:
This attribute set is only valid if both reason and time limit are defined. The Batch API will throw an error if not all attributes are defined, so a check is not strictly necessary but it would be nice to know before starting a deploy that this will fail if both are not defined.